### ISSUE TITLE
Integrate "Request" buttons for multiple levels of access

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/pulfalight#141]
Referencing the existing request buttons for the PULFA 2.0 interface, the following requests were submitted:
> It would be great to have more buttons that correspond to the various things you can do with specific material (digital or physical, open/curator review/or closed, AV that can only be accessed pending digitization etc.), as well as have the pop-up alert users when they've requested something with a restriction (or prevent them from doing so automatically). Could also be a "Could have" assuming this is pretty hard to do, but I think it would really help users navigate the site.

> Curator review should not even be an option.

> PSMudd would like a version of this option.

Based upon these requests, there would be the following levels of access:

* Digital material (born digital or digitized)
* Physical material
* Open
* Closed
* Audiovisual (pending digitization)
* Mudd Library Staff

Further, a modal dialogue should be generated for non-open access levels.

**Comment by jrgriffiniii** [comment]
This information is already available in the `accessRestrict` element: https://www.loc.gov/ead/tglib/elements/accessrestrict.html

**Comment by jrgriffiniii** [comment]
We may need to support cases where requests are pending curatorial approval.

**Comment by jrgriffiniii** [comment]
This is going to be given a lower priority given the shift in Mudd Library staff availability for the current sprint.

**Comment by regineheberlein** [comment]
current values for @type are:

'closed'
'curatorreview'
'curatorreview '
'digital'
'open'
'remotestorage'
'restricted'

**Comment by jrgriffiniii** [comment]
This is blocked by #38

**Comment by jrgriffiniii** [comment]
What is the difference between `restricted` and `curatorreview`? Is this difference needed?

@faithc will enquire about this with ADAPT, it might simply be the case that `curatorreview` was from a dated workflow which has been updated.

@regineheberlein has found that this is used only in AC028, so it might indeed just be dated.

**Comment by jrgriffiniii** [comment]
This might be moved to the 1.x release, but this still needs to be determined.

**Comment by jrgriffiniii** [comment]
This might not be necessary, as items which have `restricted` access are submitted to Aeon but rejected, while `closed` items simply won't be requestable at all.

**Comment by tpendragon** [comment]
Taken care of by OARSC.
